main.ts

```diff
@@ -628,7 +628,7 @@ class GraaspStack extends TerraformStack {
       {
         VITE_API_HOST: `https://${subdomainForEnv('api', environment)}`,
         VITE_CLIENT_HOST: `https://${envDomain(environment)}`, // apex domain
-        VITE_GRAASPER_ID: toEnvVar(graasperID),
+        VITE_CURRENT_HOST: `https://${subdomainForEnv('library', environment)}`,
```

Comment:
Could this be name `VITE_HOST` instead? Current is a bit confusing.

Reply:
yes it is better thank you